### PR TITLE
Add oreganized.json

### PR DESCRIPTION
**Pull request by Freshah**
Hope I did this correctly!  simple addition to the defaults for Lead/Silver compat.

### CODE REVIEW

File: common/src/main/resources/default_configs/oreganized.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "mod_id": "oreganized",
+  "provided_stones": [],
+  "provided_ores": [
+    {
+      "id": "lead_ore",
+      "stone": [
+        "stone",
+        "deepslate"
```
Comment (lukebemish) [inline common/src/main/resources/default_configs/oreganized.json]: Every provided stone should have a corresponding block_id.

File: common/src/main/resources/default_configs/oreganized.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      "types": [
+        "stone"
+      ]
+    },
+   
+    {
+      "id": "silver_ore",
+      "stone": [
+        "stone",
+        "deepslate"
```
Comment (lukebemish) [inline common/src/main/resources/default_configs/oreganized.json]: Same as above.

**Review by lukebemish — CHANGES_REQUESTED** [review]
Looks good; could you either provide a block ID for the deepstone variants, if they already exist, or if they don't remove it from the stone list?

**Comment by lukebemish** [comment]
If you don't mind me asking, what are the bottom left two stones in the image? (This is a bit embarrassing as I may have written a config for them)

**Comment by Freshah** [comment]
I'll fix that right up!  The stones are Shale and Jasper from Quark, yeah.

**Comment by lukebemish** [comment]
Cool! I'm glad the texture extraction works as well here as it does.

**Comment by lukebemish** [comment]
Thank you!

**Comment by Freshah** [comment]
The variants exist but don't spawn so after release I may have to update.  If I end up using more stone and ore mods I'll keep contributing.

**Comment by lukebemish** [comment]
They should spawn, though it probably depends how Oreganized is doing its ore gen and whether it's doing some really weird stuff. After it's released on 1.18.2, feel free to file a bug report if they're not spawning.

**Pull request merged**